_javascript/modules/components/toc.js

```diff
@@ -1,15 +1,29 @@
-export function toc() {
-  if (document.querySelector('main h2, main h3')) {
-    // see: https://github.com/tscanlin/tocbot#usage
-    tocbot.init({
-      tocSelector: '#toc',
-      contentSelector: '.content',
-      ignoreSelector: '[data-toc-skip]',
-      headingSelector: 'h2, h3, h4',
-      orderedList: false,
-      scrollSmooth: false
-    });
-
-    document.getElementById('toc-wrapper').classList.remove('d-none');
+import { TocMobile as mobile } from './toc/toc-mobile';
+import { TocDesktop as desktop } from './toc/toc-desktop';
+
+const mediaQuery = matchMedia('(min-width: 1200px)');
+
+function refresh() {
+  if (mediaQuery.matches) {
+    desktop.refresh();
```

Comment:
Please call `mobile.hide();` to avoid situation when viewport gets resized and we get empty popup window. I guess it will be fine to just close it since TOC will appear on the right sidepanel.

```suggestion
    mobile.hide();
    desktop.refresh();
```

Reply:
It makes sense. :)